src/GRA.Controllers/MissionControl/EmailManagementController.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Threading.Tasks;
+using GRA.Controllers.ViewModel.MissionControl.EmailManagement;
+using GRA.Controllers.ViewModel.Shared;
+using GRA.Domain.Model;
+using GRA.Domain.Model.Filters;
+using GRA.Domain.Service;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace GRA.Controllers.MissionControl
+{
+    [Area("MissionControl")]
+    [Authorize(Policy = Policy.ManageBulkEmails)]
+    public class EmailManagementController : Base.MCController
+    {
+        private readonly ILogger<EmailManagementController> _logger;
+        private readonly SiteService _siteService;
+        private readonly EmailManagementService _emailManagementService;
+        private readonly EmailService _emailService;
+
+        public EmailManagementController(ILogger<EmailManagementController> logger,
+            ServiceFacade.Controller context,
+            EmailManagementService emailManagementService,
+            EmailService emailService,
+            SiteService siteService)
+            : base(context)
+        {
+            _logger = logger ?? throw new ArgumentNullException(nameof(logger));
+            _siteService = siteService ?? throw new ArgumentNullException(nameof(siteService));
+            _emailManagementService = emailManagementService ?? throw new ArgumentNullException(nameof(emailManagementService));
+            _emailService = emailService ?? throw new ArgumentNullException(nameof(emailService));
+            PageTitle = "Email Management";
+        }
+
+        public async Task<IActionResult> Index(int page = 1)
+        {
+            var filter = new BaseFilter();
+            var templateList = await _emailManagementService.GetPaginatedEmailTemplateListAsync(filter);
+            var paginateModel = new PaginateViewModel
+            {
+                ItemCount = templateList.Count,
+                CurrentPage = page,
+                ItemsPerPage = filter.Take.Value
+            };
+            if (paginateModel.MaxPage > 0 && paginateModel.CurrentPage > paginateModel.MaxPage)
```

Comment:
I think there's a method in paginateModel called PastMaxPage or something that should be used here.

Reply:
Got it